index.bs

```diff
@@ -10361,19 +10363,12 @@ is the [=interface=]:
         [=overload resolution algorithm=].
     1.  Let |R| be the result of performing the actions listed in the description of
         |constructor| with |values| as the argument values.
+    1.  Assert: |R| is an object that implements |I|.
```

Comment:
Hmm. I think at this point |R| is a completion value, so this doesn't quite work. You'll want to add a step before the asserts that simply says "ReturnIfAbrupt(R)."

For further clarity, maybe add ", as a [=completion value=]" to the "Let R be the result..." step.